packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -42,6 +42,12 @@ final class ParallelFileProcessor
      * @var int
      */
     private const SYSTEM_ERROR_LIMIT = 50;
+    /**
+     * the number of chunks a worker can process before getting killed
+     *
+     * @var int
+     */
+    private const MAX_CHUNKS_PER_WORKER = 8;
```

Comment:
it is a bit ambiguous the different between `$jobSize` vs `MAX_CHUNKS_PER_WORKER`, could you write a comment here? Thank you.